### PR TITLE
Fix coverage report in CI

### PR DESCRIPTION
**Pull request by arcondello**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -237,13 +266,7 @@ jobs:
...
             cd tests/
             pip install -r requirements.txt
-            coverage run --source=dimod --append -m unittest
-      - run: &unix-codecov
-          name: codecov
-          command: |
-            . env/bin/activate
-            cd tests/
-            codecov
+            python -m unittest
```
Comment (randomir) [inline .circleci/config.yml:269]:
```suggestion
            python -m unittest discover
```

Comment (arcondello) [reply .circleci/config.yml:269]: Why?

Comment (randomir) [reply .circleci/config.yml:269]: Is `discover` implied? I see you use it below.

Comment (randomir) [reply .circleci/config.yml:269]: Oh, looks like it is implied. Ignore the suggestion then please.

Comment (arcondello) [reply .circleci/config.yml:269]: It is indeed. Agree I should be consistent.

**Pull request merged**